lib/utils/transformOptionsFromPath.js

```diff
@@ -0,0 +1,32 @@
+const transformOptionsFromPath = (options, relPathFromRoot) => {
+  let transformedOptions = {};
+
+  Object.keys(options).forEach(key => {
+    const transformedOpt = transformOptionFromPath(options[key], relPathFromRoot);
+    if (transformedOpt) {
+      transformedOptions[key] = transformedOpt;
+    }
+  });
+
+  return transformedOptions;
+};
+
+const transformOptionFromPath = (opt, relPathFromRoot) => {
```

Comment:
since this is a public function, probably having jsdoc would be a good idea

Reply:
? This is not public

Reply:
And jsdoc isn't used anywhere in objection codebase

Reply:
@koskimas Right, my mistake. Comment still applies, but should be at "transformOptionsFromPath" instead.

Reply:
That's not public either. Only stuff mentioned in the docs is considered public

Reply:
Fair point.